CONTRIBUTING.md

```diff
@@ -15,6 +15,9 @@ python3 -m venv venv
 ./venv/bin/pip install -e '.[dev]'
 ```
 
+####*Note:*
+Sygnal depends on the psycop database adapter for PostgreSQL. For the command above to succeed, you may first need to install the development headers for libpq, for example, by running `sudo apt install libpq-dev` on Ubuntu."
```

Comment:
Can probably get rid of the '####' prefix since we're not really introducing a new heading here; more of an aside.

Also, it's `psycopg2` not `psycop` 🙂 

I did some testing and it's possible that people will need more than just `libpq-dev`; could you please suggest folks install `libpq-dev python3-dev build-essential`?